src/AppserverIo/Appserver/ServletEngine/Utils/ErrorUtil.php

```diff
@@ -51,6 +51,7 @@ class ErrorUtil extends \AppserverIo\Appserver\Core\Utilities\ErrorUtil
      */
     public function fromException(\Exception $e)
     {
+        ErrorUtil::singleton()->flattenExceptionBacktrace($e);
```

Comment:
Instead of loading the singleton instance with `ErrorUtil::singleton()->flattenExceptionBacktrace($e)` you should use `$this->flattenExceptionBacktrace($e)`.